### PR TITLE
Improve Korean translations

### PR DESCRIPTION
**Pull request by ghost**
This PR adds and improves Korean translations more 🦊. @nujabes403 @x86kernel Would you kindly review this? 😄 

* `선택된 단위` -> `현재 통화`
* `선택 된` -> `선택된`
* `서명 중입니다` -> `서명합니다`
* `더 배우기` -> `더 알아보기`

### CODE REVIEW

File: app/_locales/ko/messages.json [hunk 1]
```diff
@@ -55,6 +64,9 @@
   "attemptingConnect": {
     "message": "블록체인에 접속을 시도하는 중입니다."
   },
+  "attemptToCancel": {
+    "message": "취소 시도하시겠습니까?"
```
Comment (nujabes403) [inline app/_locales/ko/messages.json]: "취소 시도하시겠습니까?" seems unnatural, how about changing it?

File: app/_locales/ko/messages.json [hunk 2]
```diff
@@ -907,6 +964,12 @@
   "sendTokens": {
     "message": "토큰 전송"
   },
+  "sentEther": {
+    "message": "이더 전송"
```
Comment (nujabes403) [inline app/_locales/ko/messages.json]: Shouldn't be "전송된 이더"?

File: app/_locales/ko/messages.json (hunk 1 above)
Comment (nujabes403) [reply app/_locales/ko/messages.json]: That's good :) all `취소 하시겠습니까?`

Comment (noelyoo) [reply app/_locales/ko/messages.json]: Great, let's go with `취소 하시겠습니까?` since it's shorter.

**Comment by noelyoo** [comment]
Thank you for your nice work @nujabes403 😄 
We've improved:
* `취소 하시겠습니까?`
* `전송된 이더`

**Comment by nujabes403** [comment]
LGTM!

**Review by whymarrh — APPROVED** [review]
@noelyoo @nujabes403 y'all are the best! Thanks for this, it's much appreciated!

**Pull request merged**